.evergreen/.evg.yml

```diff
@@ -316,15 +316,6 @@ functions:
           ${PREPARE_SHELL}
           echo '{"results": [{ "status": "FAIL", "test_file": "Build", "log_raw": "No test-results.json found was created"  } ]}' > ${PROJECT_DIRECTORY}/test-results.json
 
-  "install dependencies":
```

Comment:
The presence of this function caused evergreen to refuse to load the file, so had to remove it.

Reply:
Ack